mmaction/core/evaluation/eval_hooks.py

```diff
@@ -150,12 +196,34 @@ class DistEvalHook(EvalHook):
             localization dataset (ActivityNetDataset). Default: `top1_acc`.
         rule (str | None): Comparison rule for best score. If set to None,
             it will infer a reasonable rule. Default: 'None'.
```

Comment:
sorry what's the valid options for rule? totally forgot that

Reply:
‘greater’ or 'less'

Reply:
add valid options for rule in docstring

Reply:
Added last time.

Reply:
why the diff doesn't show?